docs.package.json

```diff
@@ -7,13 +7,7 @@
   },
   {
     "repo": "osism/osism.github.io",
-    "source": "docs/guides/operations-guide/openstack/image-manager.md",
-    "target": "docs/02-iaas/components",
-    "label": ""
-  },
-  {
-    "repo": "osism/osism.github.io",
-    "source": "docs/guides/operations-guide/openstack/flavor-manager.md",
+    "source": "docs/guides/operations-guide/openstack/*",
```

Comment:
Use `docs/guides/operations-guide/openstack/day2-operations/*` here. Move the files we want to include in the SCS docs at the moment there with https://github.com/osism/osism.github.io/commit/be88658bcf46e75ac190ae6913a6b95e6affb0d8.